Sofa/Sofa/Header/Sport Selector/Sports.swift

```diff
@@ -7,10 +7,10 @@
 
 import Foundation
 
-enum Sport {
-    case football
-    case basketball
-    case americanFootball
+enum Sport: String, CaseIterable {
+    case football = "football"
+    case basketball = "basketball"
```

Comment:
```suggestion
    case football
    case basketball 
```

Kad dodaš da se po Stringu radi rawValue, automatski se ime casea definira u String.